src/api/spec/cassettes/Cloud_Backend_UploadJob/_all/with_an_invalid_backend_response/1_2_2_1.yml

```diff
@@ -30,7 +30,7 @@ http_interactions:
       encoding: UTF-8
       string: |
         <status code="400">
-          <summary>no cloud upload server configurated</summary>
+          <summary>no cloud upload server configured</summary>
         </status>
     http_version: 
   recorded_at: Thu, 04 Jan 2018 12:28:11 GMT
```

Comment:
so you just replaced the string in the cassettes? Didn't dare to regenerate them? Because I had quite some problems with this test without cassettes.

Reply:
Because you have an old BSConfig.pm... :)

Reply:
Quick and dirty, yes :smile: 
I can run them but I don't want to spend too much time just for fixing a typo :smile: If it fails, I will close this PR for now ...

Reply:
> If it fails, I will close this PR for now ...

What I mean: I understand that the preferred way should be to recreate the tests. But I don't want to invest ~1 hour fixing the cassettes so I just replaced the string also in the cassettes. If there is a consent to run the cassettes again, I will close this PR and create an issue.

Reply:
The test runs fine with a generated BSConfig.pm